### ISSUE TITLE
08/30/2022 - SIG UX-UI - Year end meeting. What's next for SIG-UI-UX

### ISSUE THREAD

**Issue by rainbj** [o3de/sig-ui-ux#81]
## UI-UX SIG Meeting — todays content for 08/30/2022

Hi O3DE Community,
We want to invite you to our conversation about today's newest content, design specifications, and program update. 

**This weeks conversation:**  
 
    RET updates
    Postmortem
    Re-elections (should we re-run?) @aFinchy talk through the details
    How should we reschedule meetings for the upcoming year (monthly?)

 
 
 ## Schedule
- **Date/Time:** 08/30/2022 @ 10:00am Pacific Time (PT) 
- **Location:** [Discord SIG-UI-UX Voice Room](https://discord.gg/Mc6jStmuMK)
- **Moderator:** UI/UX SIG Committee
- **Note Taker** UI/UX SIG Committee

## Meeting Details
We want to invite you to our Sig UX UI expert interviews, global updates, Roadmap visibility, and design review. This meeting is bi-weekly and we will cover the following items:

•  New and interesting topics that O3DE has hot of the press. Because these items are so new we are likely to find bugs and new feedback from the community. This is kind of the point. As we let you observe and talk to us we can make adjustments to the workflows before it goes out to the masses.  
• RTE program updates and new details. Any new findings, results, or feedback.  
• Check in with designer about problems they are facing regarding tickets and if any help is required.  
• Feedback from any research previously done.  
• Assign designers or UI-UX SIG members UX work.  
• Conduct live research.  
• Create any ticket or close tickets based on our conversation.  

## Past meetings, including a link to the agenda, recording, notes, and resources.
 https://github.com/o3de/sig-ui-ux/issues?q=is%3Aissue+is%3Aclosed+todays+in%3Atitle

## To sign up for a topic in the Todays topic Bi-weekly meeting. 
 - Please add a **comment** to this ticket with following information
   - Discord name.
   - Topic of conversation.  
   - How much time you might need.  
   - Any UX resource you have already been working with and if their attendance is required.  
   - Any resource to help explain your issue. 

## How can I get UX help on an issue?
We generally use the alternative weeks open office hours meeting to have conversations like this. Please look for the next meeting here and include a request in the comments. As long as time is open we will address your agenda right away. Please give us 24 hours notice before the meeting to have a UX expert in that area join the meeting. 
 https://github.com/o3de/sig-ui-ux/issues?q=is%3Aissue+is%3Aopen+Office+in%3Atitle+
  
 
## How can I Help? 
• Great question. If you have a speciality such as research, design, or a unique understanding of game development. Let us know so we can help facilitate conversation on how best to contribute.  
• Offer to be a note taker. We always need some help here. Facilitating a meeting and being a note taker can be tough, so any one willing to help will be appreciated.   
• Offer to be a person willing to help put together agenda items and or review agenda items from previous meetings.  

## What should I bring to the bi-weekly meeting?
• Nothing, unless your a presenter, your attendance and great conversation is enough

## Cut-off time / cancellation 
• For the UI/UX bi-weekly meeting the UI-UX SIG team will check the GitHub Agenda item ticket 24 hours before the scheduled meeting, if no one signs up in the session, we will lock down the agenda of that day.
• If you are unable to attend after signing up, Please just reach out in discord or on the ticket to let us know about the change.

**Comment by rainbj** [comment]
We are going to review the following. Please feel free to add any additional topics>

- RET updates
- Postmortem
- Re-elections (should we re-run?) @aFinchy talk through the details
- How should we reschedule meetings for the upcoming year (monthly?)

**Comment by yuyihsu** [comment]
I'd also like to talk about the roadmap for sig ui-ux

**Comment by rainbj** [comment]
08/30 SIG UI-UX meeting recap notes.
•	During our SIG meeting, we spoke about our upcoming SIG-UI-UX election process. We brought in Amy, our content manager, and we talked about the election process, the guidelines and the requirements. She explained that this process is a mandatory process, and it needs to be run once a year. However, the election process only happens if there are more than two people, interested in the job. The link below is the guidelines on how this process should be implemented. From a Linux foundation side, it needs to be shown that we have committed to renewing or transfer ownership yearly. 
o	https://github.com/o3de/community/blob/main/O3DE%20Elections/O3DE%20Elections%20Guide.md 
o	As for timeline on when this should get done. We were informed that it’s okay to have elections stagger throughout the year to prevent overlap with other SIG elections. Amy also said she would be willing to help with announcements for our election. This includes creating an event and making announcements on our digital channels. 
o	The next step for us will be to create a page based on the SIG release version and post it as a new ticket inside the SIG-UI-UX channel. An example can be seen here. https://github.com/o3de/sig-release/issues/74
•	The next meeting topic was updating this ongoing meeting. (new content, triage, and RTE). The concern is that this meeting has become less effective than it used to be as our meeting has been broken down into smaller sub meetings. The team wants to find ways to use this time more effectively. Some of the points of the conversation were as followed. 
o	If we gave more time between meetings, it would allow for more people to list topics of importance, it would also allow for better planning and higher attendance. 
o	At this moment, this meeting is a by weekly meeting, maybe we should have this one monthly? 
o	For the SIG-UI-UX group, we also have a weekly office hours/triage meeting. That meeting feels more purposeful. How can we align with that type of purposeful intent? 
o	We are noticing the low attendance on this meeting. Should we be concerned about attendance as a factor for running the SIG meetings? Is that a relevant factor?
o	Should we be encouraging people to attend more purposeful agenda meeting types with more impactful topics?
o	Should this meeting be moved to a less structured meeting type and create a more open format for meeting, such as the “Water cooler talk” format. Where no one is running the meeting, but it's an active time to just chat with the community about any unstructured conversation about O3DE. This could even host what people are working on without some kind of structured format.
o	We could also follow the suggested format of the SIG’s doc community, just (triage and office hours)
o	JT said he likes open office hours and some of the free format meeting styles. He likes when people can come and talk and show whatever topics you want.
•	Yuyi showed off some of the survey results from the most recent RTE results. This includes some of the overall scores and individual scores per categories. Including diving into overall scores and updates score coming in from WF6.
